packages/waci-pex/__tests__/local-agent.test.tsx

```diff
@@ -0,0 +1,27 @@
+import {getConfig} from '@veramo/cli/build/setup'
```

Comment:
please keep file names the same as we use in every other plugin.

Reply:
updated

Reply:
you updated some files, not all of them. You need to check all the files